README.md

```diff
@@ -84,11 +85,35 @@ https://unpkg.com/prismic-javascript
 
 On our release page: [https://github.com/prismicio/prismic-javascript/releases](https://github.com/prismicio/prismic-javascript/releases).
 
-The kit is universal, it can be used:
 
-* Server-side with NodeJS
-* Client-side as part of your build with Browserify, Webpack
-* Client-side with a simple script tag
+#### Files
+
++ prismic-javascript.js: CommonJs2 build for Node.js 
++ prismic-javascript.mjs: ESM build for Node.js and code compilers
++ prismic-javascript.browser.js: UMD build for browsers
++ prismic-javascript.browser.mjs: ESM build for browsers
+
+#### Polyfills
+To support legacy browsers include a fetch and promise polyfills.
+For node environments include a fetch polyfill.
+
+#### Bundling 
+Webpack should work as is.
+If using Rollup remember and targeting browsers remember to turn on the browser flag in the [resolver options](https://github.com/rollup/plugins/tree/master/packages/node-resolve#browser) 
```

Comment:
Small typo: the first remember should not be here.

Reply:
Thanks, fixed it :)